docs/src/components/pricing/PricingTable.tsx

```diff
@@ -737,8 +737,8 @@ const premiumData: Record<string, React.ReactNode> = {
   'mui-x-development': <Info value="1 year" />,
   'mui-x-updates': <Info value="1 year" />,
   // Support
-  community: yes,
-  'bugs/features': <Info value={yes} metadata="Priority over Pro" />,
+  'core-support': <Info value={pending} metadata="Priority add-on" />,
```

Comment:
Match the other cells

```suggestion
  'core-support': <Info value={pending} metadata="priority add-on only" />,
```